core/src/main/java/org/mskcc/cbio/portal/scripts/ImportStructuralVariantData.java

```diff
@@ -36,134 +35,46 @@
-                structuralVariant.setTumorVariantCount(TabDelimitedFileUtil.getPartInt(getColumnIndex(TUMOR_VARIANT_COUNT), parts));
-                structuralVariant.setNormalPairedEndReadCount(TabDelimitedFileUtil.getPartInt(getColumnIndex(NORMAL_PAIRED_END_READ_COUNT), parts));
-                structuralVariant.setTumorPairedEndReadCount(TabDelimitedFileUtil.getPartInt(getColumnIndex(TUMOR_PAIRED_END_READ_COUNT), parts));
-                structuralVariant.setNormalSplitReadCount(TabDelimitedFileUtil.getPartInt(getColumnIndex(NORMAL_SPLIT_READ_COUNT), parts));
-                structuralVariant.setTumorSplitReadCount(TabDelimitedFileUtil.getPartInt(getColumnIndex(TUMOR_SPLIT_READ_COUNT), parts));
-                structuralVariant.setAnnotation(TabDelimitedFileUtil.getPartString(getColumnIndex(ANNOTATION), parts));
-                structuralVariant.setBreakpointType(TabDelimitedFileUtil.getPartString(getColumnIndex(BREAKPOINT_TYPE), parts));
-                structuralVariant.setCenter(TabDelimitedFileUtil.getPartString(getColumnIndex(CENTER), parts));
-                structuralVariant.setConnectionType(TabDelimitedFileUtil.getPartString(getColumnIndex(CONNECTION_TYPE), parts));
-                structuralVariant.setEventInfo(TabDelimitedFileUtil.getPartString(getColumnIndex(EVENT_INFO), parts));
-                structuralVariant.setVariantClass(TabDelimitedFileUtil.getPartString(getColumnIndex(VARIANT_CLASS), parts));
-                structuralVariant.setLength(TabDelimitedFileUtil.getPartInt(getColumnIndex(LENGTH), parts));
-                structuralVariant.setComments(TabDelimitedFileUtil.getPartString(getColumnIndex(COMMENTS), parts));
-                structuralVariant.setExternalAnnotation(TabDelimitedFileUtil.getPartString(getColumnIndex(EXTERNAL_ANNOTATION), parts));
-                structuralVariant.setDriverFilter(TabDelimitedFileUtil.getPartString(getColumnIndex(DRIVER_FILTER), parts));
-                structuralVariant.setDriverFilterAnn(TabDelimitedFileUtil.getPartString(getColumnIndex(DRIVER_FILTER_ANNOTATION), parts));
-                structuralVariant.setDriverTiersFilter(TabDelimitedFileUtil.getPartString(getColumnIndex(DRIVER_TIERS_FILTER), parts));
-                structuralVariant.setDriverTiersFilterAnn(TabDelimitedFileUtil.getPartString(getColumnIndex(DRIVER_TIERS_FILTER_ANNOTATION), parts));
+                if (!structuralVariantUtil.hasRequiredStructuralVariantFields(structuralVariant)) {
+                    ProgressMonitor.logWarning("Invalid Site 1 or 2 Ensembl transcript ID or exon found, ignoring structural variant for SV record #" +
+                            recordCount + " (sample, site 1 gene, site 2 gene):  (" +
+                            structuralVariant.getSampleId() + ", " + structuralVariant.getSite1HugoSymbol() +
+                            ", " + structuralVariant.getSite2HugoSymbol() + ")");
+                    continue;
```

Comment:
it might be better rearranging if else to remove "continue" 
```
                if (structuralVariantUtil.hasRequiredStructuralVariantFields(structuralVariant)) {
                     // get sample
                     Sample sample = DaoSample.getSampleByCancerStudyAndSampleId(
                      ....
                }
                else {
                    ProgressMonitor.logWarning("Invalid Site 1 or 2 Ensembl transcript ID or exon found, ignoring structural variant for SV record #" +
                            recordCount + " (sample, site 1 gene, site 2 gene):  (" +
                            structuralVariant.getSampleId() + ", " + structuralVariant.getSite1HugoSymbol() +
                            ", " + structuralVariant.getSite2HugoSymbol() + ")");
                }            
```

Reply:
```
if (...) {
continue
}
```

looks cleaner in my opinion but I think it's just a matter of personal preference. I don't believe there really is any consistency across the codebase.

Reply:
it is okay go with your way. I normally do not use a jump (continue) in the middle of code.